src/tests/utils/index.ts

```diff
@@ -62,6 +62,27 @@ export function createERC20Contract(
   return new Contract(address, abi, providerOrSigner)
 }
 
+export async function approveErc20(
+  address: string,
```

Comment:
@janndriessen Because "address" is such a common keyword, it's helpful to be more specific. Something like `erc20address`